apps/src/code-studio/pd/workshop_enrollment/workshop_enroll.jsx

```diff
@@ -41,13 +42,36 @@ export default class WorkshopEnroll extends React.Component {
   constructor(props) {
     super(props);
 
+    if (this.props.workshop.course === 'Build Your Own Workshop') {
```

Comment:
is there a way we can use a constant here? If not, maybe that's something we can tackle in a follow-up?